model/schema/variation.yaml

```diff
@@ -182,16 +193,16 @@ slots:
     exact_mappings:
       - SO:0001889
 
-  dna_mutation_type:
+  dna_mutation_type: # Should the range of this be an object (SOTerm / OntologyTerm) rather than just a curie?
```

Comment:
@cmpich @ybradford 
Are the values for this field a ZFIN-specific vocabulary? I see, for example, on these pages:
http://zfin.org/ZDB-ALT-211027-5
Type            Allele with one deletion
http://zfin.org/ZDB-ALT-220726-6
Type            Allele with one insertion
Are these SO Terms or a CV or something else?